modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ModelValidationUtils.java

```diff
@@ -63,4 +65,13 @@ static void validateConfigParametersNamesNotAllowed(ParameterizedModel model, Pr
                                                 NAME_ATTRIBUTE_NAME))));
   }
 
+  static boolean overrideEqualsAndHashCode(Class<?> clazz) {
+    try {
+      return (!clazz.getMethod("equals", Object.class).getDeclaringClass().equals(Object.class))
+          && (!clazz.getMethod("hashCode").getDeclaringClass().equals(Object.class));
+    } catch (NoSuchMethodException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS00108))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7618&issues=AWhxwX0HotIXnmsUr_SA&open=AWhxwX0HotIXnmsUr_SA)

[ISSUE_KEY]: AWhxwX0HotIXnmsUr_SA